{
  "schema_version": "1.5.0",
  "id": "CVE-2019-19911",
  "details": "There is a DoS vulnerability in Pillow before 6.2.2 caused by FpxImagePlugin.py calling the range function on an unvalidated 32-bit integer if the number of bands is large. On Windows running 32-bit Python, this results in an OverflowError or MemoryError due to the 2 GB limit. However, on Linux running 64-bit Python this results in the process being terminated by the OOM killer.",
  "aliases": [
    "CVE-2019-19911"
  ],
  "affected": [
    {
      "packages": [
        {
          "ecosystem": "bitnami",
          "name": "pillow",
          "purl": "pkg:bitnami/pillow"
        }
      ],
      "severity": [
        {
          "type": "CVSS_V3",
          "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
        }
      ],
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "6.2.2"
            }
          ]
        }
      ]
    }
  ],
  "database_specific": {
    "severity": "High",
    "cpes": [
      "cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*"
    ]
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3DUMIBUYGJRAVJCTFUWBRLVQKOUTVX5P/"
    },
    {
      "type": "WEB",
      "url": "https://pillow.readthedocs.io/en/stable/releasenotes/6.2.2.html"
    },
    {
      "type": "WEB",
      "url": "https://usn.ubuntu.com/4272-1/"
    },
    {
      "type": "WEB",
      "url": "https://www.debian.org/security/2020/dsa-4631"
    }
  ],
  "published": "2023-05-30T11:24:50.496Z",
  "modified": "2023-05-30T11:24:50.496Z"
}